openid-federation-1_0.xml

```diff
@@ -6158,7 +6180,9 @@ redirect_uri=https%3A%2F%2Frp.example.com%2Fauthz_cb
 	    If the OP fails to establish trust with the RP,
 	    it MUST treat the redirection URI as invalid
 	    and not perform redirection.
-	    This means that the two error codes defined below,
+	    This means that the error codes
+	    <spanx style="verb">invalid_trust_anchor</spanx> and
+	    <spanx style="verb">invalid_trust_chain</spanx>
 	    both of which are about reasons trust failed to be established,
```

Comment:
The new `invalid_metadata` error falls into the same category. The OP couldn't register the RP metadata because of faulty metadata (or policy conflict).